packages/storybook/stories/IdentityManagement/IdentityManagement.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Canvas, Meta, Source, Story } from '@storybook/addon-docs/blocks';
+import * as stories from './IdentityManagement.stories';
+
+import CallBackendText from '!!raw-loader!./snippets/CallBackend.snippet.ts';
+import ChatBackendText from '!!raw-loader!./snippets/ChatBackend.snippet.ts';
+
+<Meta id="identitymanagement" title="Identity Managemment" />
+
+# Identity Management
+
+It is a primary tenet of Azure Communication Services that customers bring their own user identities. Customers
```

Comment:
I suggest breaking it down into steps. For example,
- Customers deploy a secure services for generating tokens
- Services  create tokens using Azure... identity services.
- These tokens are supplied to applications....

etc etc...

Reply:
Can you link to the source of this information? `customers bring their own user identities` sounds misleading.

Reply:
Linked to the source in the docs, and reworded to be similar to the starting paragraph there.

Reply:
Also moved to bulleted list.